taskchampion/lib/src/server.rs

```diff
@@ -180,15 +182,27 @@ pub unsafe extern "C" fn tc_server_new_gcp(
             //  - bucket ownership is transferred to this function
             let bucket = unsafe { TCString::val_from_arg(bucket) }.into_string()?;
 
+            // SAFETY:
+            //  - credential_path is valid (promised by caller)
+            //  - credential_path ownership is transferred to this function
+
+            let credential_path =
+                unsafe { TCString::val_from_arg(credential_path_argument) }.into_string()?;
+            let credential_path = if credential_path == "" {
```

Comment:
I think clippy is suggesting using `.is_empty()` instead of comparing to `""` here.

Reply:
Okay!